google_drive/google_calendar.py

```diff
@@ -42,7 +42,7 @@ class GoogleCalendarBlock(XBlock, PublishEventMixin):
 
     views = [(0, 'Week'), (1, 'Month'), (2, 'Agenda')]
 
-    def student_view(self, context={}):
+    def student_view(self, context):
```

Comment:
Workbench had a problem with this part of the code, as if it didn't set context to an empty dictionary, thus throwing an error that a None object doesn't have a method .update().